### PR TITLE
Fewer featured publcations

### PR DESCRIPTION
**Pull request by lokijuhy**

### CODE REVIEW

File: content/publication/wang-2020-muc16/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Results: Analysis results showed that samples with MUC16 mutations had a higher
 
 Conclusions: MUC16 mutation appears to be a useful predictive marker of global TMB and patient survival in melanoma."
 publication: "Cancer Epidemiol Biomarkers Prev"
-featured: true
+featured: false
```
Comment (orisenbazuru) [inline content/publication/wang-2020-muc16/index.md:29]: I would keep this in featured.

File: content/publication/cruz-2020-PARP1/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ authors:
 - Joann B. Sweasy
 publication_types: ["3"]
 abstract: "Poly-ADP-ribose polymerase (PARP) inhibitors are active against cells and tumors with defects in homology-directed repair as a result of synthetic lethality. PARP inhibitors have been suggested to act by either catalytic inhibition or by PARP localization in chromatin. In this study, we treat human HCC1937 BRCA1 mutant and isogenic BRCA1-complemented cells for three weeks with veliparib, a PARP inhibitor. We show that long-term treatment with veliparib results in chromatin-bound PARP1 in the BRCA1 mutant cells, and that this correlates with significant upregulation of inflammatory genes and activation of the cyclic GMP–AMP synthase (cGAS)/ signalling effector stimulator of interferon genes (STING) pathway. In contrast, long-term treatment of isogenic BRCA1-complemented cells with veliparib does not result in chromatin-associated PARP or significant upregulation of the inflammatory response. Our results suggest that long-term veliparib treatment may prime BRCA1 mutant tumors for positive responses to immune checkpoint blockade."
-featured: True
+featured: false
```
Comment (orisenbazuru) [inline content/publication/cruz-2020-PARP1/index.md:28]: I would also keep this in featured!

**Review by orisenbazuru — COMMENTED** [review]
I would not mind keeping last 9 publications in featured. But if you want to reduce, I think keeping 5 is OK. I added comments for ones that can be left in featured (mainly Bioinformatics/medicine research). At some point it will be difficult to choose what is left in featured if we get many publications -- maybe one option is to keep the latest 5 publications by date of release! (or random choice from list of publications in a current year :)

**Comment by lokijuhy** [comment]
@orisenbazuru Can you please discuss this with Michael and come to a decision with him? I personally don't care either way. Another option Michael mentioned was reducing or removing the abstracts.

**Comment by orisenbazuru** [comment]
Collapsing the abstracts is a good idea (by default you show title and authors with abstract collapsed - you would need to press to uncollapse)

**Comment by lokijuhy** [comment]
Implementing a collapsed abstract feature would take me quite some time to figure out how to modify the website. Another option that would be easier to implement would be to use the publication "Summary" field. If Summary is filled out, the website displays that on the featured publication card instead of the whole abstract. You can see that working on the AutoDiscern publication. Could you write 1-2 sentence summaries for the other four publications you'd like to be featured?

**Comment by orisenbazuru** [comment]
I am ok with featuring less or removing the abstract (maybe you could try to have summary filled with link to publication or "space" such that it might just pick that up when displaying in the front page. Feel free to proceed with whatever you think is best/practical at this point.

**Comment by lokijuhy** [comment]
I think the best strategy is to write 1-2 sentence summaries in the `summary:` field to take the place of the abstracts. Could you please do this? 

There are 4 publications that need summaries (Michael requested 2, and the 2 you marked above). These 4 are:
* Predicting base editing outcomes with an attention-based deep learning algorithm trained on high-throughput target library screens
* Patient Similarity Analysis with Longitudinal Health Data
 * wang-2020-muc16
* cruz-2020-PARP1

**Comment by lokijuhy** [comment]
@orisenbazuru I'm merging this PR as is, since that was Michael's request to me. If you'd like to add back the other 2 publications, in a separate PR, please write 1-2 sentence `summary` entries for them so they don't take up so much space on the screen, as I described in my previous comment.

**Pull request merged**